packages/api/src/controllers/user.ts

```diff
@@ -296,7 +296,11 @@ app.delete("/:id", authorizer({ anyAdmin: true }), async (req, res) => {
     res.status(404);
     return res.json({ errors: ["Account not found"] });
   }
-  await db.user.delete(id);
+
+  await db.user.update(id, {
+    deleted: true,
+    deletedAt: Date.now(),
+  });
```

Comment:
I don't think it is that simple. We have several places that read the user object which don't check either of these variables. I believe we should either keep it as a hard-delete or do this big refactor fixing these places.

Reply:
Yeah, it was opened in a rush, but let's close it and I focus on making a proper change to deleted all the streams + assets + sessions when a user is hard deleted. 

We may also consider to delete all the contents but keep the user row redacting all the personal info to avoid userIds pointing to non exsiting users, wdyt? Is it GDPR compliant?